lib/decoder.js

```diff
@@ -213,6 +224,10 @@ Decoder.prototype.decode = function(len, data, callback) {
     }
   }
 
+  if(len === 11) {
```

Comment:
That's also what other implementations are doing, having a lookup table rather than "guessing", but I saw the length-based approach, so I implemented it.
The problem I have with the current implementation is the conversion to string:
```
if(type === 'UNKN') {
  value = data.toString();
 }
```
If we could output this in hex, one could also work with non-printable characters.